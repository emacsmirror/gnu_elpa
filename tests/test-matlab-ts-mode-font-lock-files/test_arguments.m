% -*- matlab-ts -*-
function test_arguments(in1, properties, options)
    arguments
       in1 (1,:) {mustBeNumeric}
       properties double
       options.LineStyle (1,1) string = "-" 
       options.LineWidth (1,1) {mustBeNumeric} = 1
    end
end
