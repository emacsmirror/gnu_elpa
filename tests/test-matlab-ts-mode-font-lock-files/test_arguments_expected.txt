c ccc ccccccccc ccc
kkkkkkkk ffffffffffffffbvvvD vvvvvvvvvvD vvvvvvvb
    kkkkkkkkk
       vvv ttttt ttttttttttttttt
       vvvvvvvvvv tttttt
       vvvvvvvvvvvvvvvvv ttttt tttttt t ttt 
       vvvvvvvvvvvvvvvvv ttttt ttttttttttttttt t t
    kkk
kkk
